app/views/api/v2/compute_resources/available_resource_pools.rabl

```diff
@@ -1,3 +1,3 @@
 collection @available_resource_pools
 
-attribute :name, :id
+attribute :name, :id, :cluster, :datacenter, :overall_status
```

Comment:
I have got no idea, so asking if we know, or are able to find out: how is overall_status calculated, will it not infer some heavy calculation on the VmWare side?

Reply:
Let me remove it, the more I think about it, we just want to show cluster info mostly for other commands, so something like status would better be seen in the actual vsphere client.